packages/web-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+
+/** RadioGroup styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    align-items: flex-start;
+    flex-direction: column;
+    margin: 2px 0;
+    row-gap: ${spacingVerticalS};
+  }
+  ::slotted([slot='label']) {
+    color: ${colorNeutralForeground3};
+    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
+    font: ${fontWeightRegular} ${fontSizeBase300} / ${lineHeightBase300} ${fontFamilyBase};
+    cursor: default;
+  }
+
+  .positioning-region {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  :host([orientation='vertical']) .positioning-region {
+    flex-direction: column;
+  }
+
+  :host([orientation='horizontal']) .positioning-region {
+    flex-direction: row;
+  }
+
+  :host([orientation='horizontal'][stacked]) ::slotted([role='radio']) {
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  :host([disabled]) ::slotted([role='radio'] ::slotted(.label)) {
```

Comment:
Where is the `.label` class coming from here? We shouldn't sprout classes on host elements, so I don't think I'd anticipate this being on a given element. If the thinking is that this needs to be added by a consumer I'm not sure that's a great ergonomic

Reply:
`.label` class comes directly from the FAST Radio template.

![image](https://user-images.githubusercontent.com/11062709/224866949-216c6121-ef0f-4de3-9f10-3153a6148556.png)

Reply:
That’s in the shadow root, is this actually reaching into the shadow root and grabbing it and applying it? The label text slotted into the radio is going to be in the light dom but even an element would only include selectors targetable by this in the light dom.

Reply:
Oh wait I'm sorry I see what you are looking at now. Yeah that's dead code it never worked and shouldn't have been commited. I was just experimenting to see if it was possible to reach into the shadow DOM of a child component as a potential solution to being able to style `Radio` slotted content when the `RadioGroup` is `disabled`.

Reply:
Follow up.. the radio's label also needs specific styling when `stacked="true"`  is set on the `RadioGroup`.